app/static/js/userProfile.js

```diff
@@ -158,7 +158,29 @@ $(document).ready(function(){
         }
     });
 
-    setupPhoneNumber("#updatePhone", "#phoneInput")
+  $("#updatePhone").on('click', function() {
+    var username = $(this).data("username")
+    validatePhoneNumber(this, "#phoneInput", username)
+  });
```

Comment:
This is old code that was carried over. It breaks the phone number handler.

Reply:
deleted,